rotkehlchen/tests/unit/decoders/test_weth.py

```diff
@@ -465,3 +490,581 @@ def test_wxdai_wrap(database, gnosis_inquirer, gnosis_accounts):
+        database=database,
+        tx_hash=evmhash,
+    )
+    timestamp = TimestampMS(1712238368000)
+    amount, gas_fees = '0.00052', '0.00000108547'
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label='0xBE6660FBE96B61B72Bf35FFaB40eB2CA886A7f85',
+            notes=f'Burned {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ),
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+            asset=A_WETH_ARB,
+            balance=Balance(amount=FVal(amount)),
+            location_label='0xBE6660FBE96B61B72Bf35FFaB40eB2CA886A7f85',
+            notes=f'Unwrap {amount} WETH',
+            counterparty=CPT_WETH,
+            address=string_to_evm_address('0x82aF49447D8a07e3bd95BD0d56f35241523fBab1'),
+        ),
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(amount)),
+            location_label='0xBE6660FBE96B61B72Bf35FFaB40eB2CA886A7f85',
+            notes=f'Receive {amount} ETH',
+            counterparty=CPT_WETH,
+            address=string_to_evm_address('0x82aF49447D8a07e3bd95BD0d56f35241523fBab1'),
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x7aBAee8F04EFd689961115f7A28bAA2E73Be6703']])
+def test_weth_deposit_arbitrum_one(database, arbitrum_one_inquirer):
+    evmhash = deserialize_evm_tx_hash('0x57cc837c6f3d84c8fa3db8a7405f7244f11d32152159edf5ba79f5a7c34919b8')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    timestamp = TimestampMS(1712328694000)
+    amount, gas_fees = '0.007767825959188763', '0.000001382891214'
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label='0x7aBAee8F04EFd689961115f7A28bAA2E73Be6703',
```

Comment:
same here as the previous test. And I guess for all rest in this file and other files you made/touched